udata/core/organization/views.py

```diff
@@ -97,77 +97,8 @@ def get_context(self):
 
 
 @blueprint.route('/<org:org>/dashboard/', endpoint='dashboard')
-class OrganizationDashboardView(OrgView, DetailView):
-    template_name = 'organization/dashboard.html'
-
-    def get_context(self):
-        context = super(OrganizationDashboardView, self).get_context()
-
-        widgets = []
-
-        if self.organization.metrics.get('datasets', 0) > 0:
-            widgets.append({
-                'title': _('Datasets'),
-                'widgets': [
-                    {
-                        'title': _('Datasets'),
-                        'metric': 'datasets',
-                        'type': 'line',
-                        'endpoint': 'datasets.list',
-                        'args': {'org': self.organization}
-                    },
-                    {
-                        'title': _('Views'),
-                        'metric': 'dataset_views',
-                        'data': 'datasets_nb_uniq_visitors',
-                        'type': 'bar',
-                        'endpoint': 'datasets.list',
-                        'args': {'org': self.organization}
-                    }
-                ]
-            })
-
-        if self.organization.metrics.get('reuses') > 0:
-            widgets.append({
-                'title': _('Reuses'),
-                'widgets': [
-                    {
-                        'title': _('Reuses'),
-                        'metric': 'reuses',
-                        'type': 'line',
-                        'endpoint': 'reuses.list',
-                        'args': {'org': self.organization}
-                    },
-                    {
-                        'title': _('Views'),
-                        'metric': 'reuse_views',
-                        'data': 'reuses_nb_uniq_visitors',
-                        'type': 'bar',
-                        'endpoint': 'reuses.list',
-                        'args': {'org': self.organization}
-                    }
-                ]
-            })
-
-        widgets.append({
-            'title': _('Community'),
-            'widgets': [
-                {
-                    'title': _('Permitted reuses'),
-                    'metric': 'permitted_reuses',
-                    'type': 'line',
-                },
-                {
-                    'title': _('Followers'),
-                    'metric': 'followers',
-                    'type': 'line',
-                }
-            ]
-        })
-
-        context['metrics'] = widgets
-
-        return context
+def organization_dashboard(org):
+    return redirect('%s#dashboard' % url_for('.show', org=org), code=301)
```

Comment:
Agreed for the consistency.

The dot-something was found here: http://flask.pocoo.org/docs/0.12/blueprints/#building-urls

Although it seems magical to me the route knows in which context the route is.